content/en/getting-started/help-and-support/index.md

```diff
@@ -10,89 +10,120 @@ cascade:
 
 ## Introduction
 
-We strive to make it as easy as possible for you to use LocalStack, and we are very grateful for any feedback.
-We provide different levels of support to help you with your queries and issues.
-The support you receive depends on the plan you are on.
+The support team is available to help you with:
```

Comment:
This intro reads a little weird. First, it is talking about "official" support but hasn't actually laid out who gets that yet. For example, you can definitely ask questions about general troubleshooting in the community Slack (no idea if anyone will answer but that's a different issue of building an active community that we aim to solve).

It also tells you support doesn't answer billing or non-technical issues but then tells if you have those kinds of issues email support.

I would start by laying out (as you did in the prior version) that there are a variety of support channels available to LocalStack users and that all users, even free ones, have access to support. You access to certain types of support channels depend upon which type of account/plan you have. Something like that. The message should be something like we're here to help but right now the intro reads very terse and unfriendly to me.